src/main/features/core/tray.js

```diff
@@ -31,21 +36,37 @@ let audioDeviceMenu = [
   },
 ];
 
-appIcon = new Tray(path.resolve(trayNormalPath, appIconFileName));
+setAppIconFileName();
+
+appIcon = new Tray(appIconFileName);
+
+Emitter.on('settings:set', (event, details) => {
+  if (details.key === 'appIconInvert') {
+    appIconInvert = details.value;
+    setAppIconFileName();
+    appIcon.setImage(appIconFileName);
+  }
+});
 
 // Change the icon if the music is playing
 Emitter.on('playback:isPlaying', () => {
-  appIcon.setImage(path.resolve(trayPlayingPath, appIconFileName));
+  currentIconPath = trayPlayingPath;
+  setAppIconFileName();
```

Comment:
Purely to make this look nicer, why don't we make `setAppIconFileName()` return `appIconFileName` so these two lines can be simplified to.

`appIcon.setImage(setAppIconFileName())`